package.json

```diff
@@ -21,17 +21,21 @@
   },
   "devDependencies": {
     "@types/node": "^14.14.8",
+    "@types/redis2": "npm:@types/redis@^2.8.28",
     "@typescript-eslint/eslint-plugin": "^4.8.1",
     "@typescript-eslint/parser": "^4.8.1",
     "ava": "^3.13.0",
     "eslint": "^7.13.0",
     "eslint-config-prettier": "^6.15.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "ioredis": "^5.4.1",
+    "node": "^18.20.4",
```

Comment:
You don't install note via package.json, node and npm need to be installed before you can use the package.json

look in the CI workflow file

Reply:
oh whoops thats right, forgot about that. Then should I change 12 => in the npm-publish.yml file instead (once I reverse the last commit)
```
      - uses: actions/setup-node@v1
        with:
          node-version: 12 =>18
```